testData/solutions/win32.json5

```diff
@@ -675,56 +828,129 @@
                             "default": 1
                         }
                     },
-                    "HTML.MaxLineLength": {
+                    "HTML.IdentifySamePageLinks": {
                         "schema": {
-                            "title": "Max Line Length",
-                            "description": "The maximum number of characters which can appear on a line of a paragraph.",
-                            "type": "integer",
-                            "default": 150
+                            "title": "Identify Same Page Links",
+                            "description": "Whether to identify links that point to other places on the current page by saying \"same page link\".",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "off",
+                                "on"
+                            ],
+                            "default": 1
                         }
                     },
-                    "HTML.PageRefreshFilter": {
+                    "HTML.LinkText": {
                         "schema": {
-                            "title": "Page Refresh Filter",
-                            "description": "How often, in milliseconds, to allow all refreshes.  Set to 0 to allow all refreshes without any delay.",
-                            "type": "integer",
-                            "default": 0
+                            "title": "",
```

Comment:
I'm gonna have to harden up the title/description check, I see.  Obviously an empty string isn't appropriate.

Reply:
I've commented out the settings that have empty titles.